forge-game/src/main/java/forge/game/ability/effects/DamageDealEffect.java

```diff
@@ -288,18 +283,7 @@ protected void internalDamageDeal(SpellAbility sa, Card sourceLKI, Card c, int d
         int excess = 0;
         int dmgToTarget = 0;
         if (sa.hasParam("ExcessDamage") || sa.hasParam("ExcessSVar")) {
-            int lethal = 0;
-            if (c.isCreature()) {
-                lethal = Math.max(0, c.getLethalDamage());
-                if (sourceLKI.hasKeyword(Keyword.DEATHTOUCH)) {
-                    lethal = Math.min(lethal, 1);
-                }
-            }
-            if (c.isPlaneswalker()) {
-                int lethalPW = c.getCurrentLoyalty();
-                // 120.4a
-                lethal = c.isCreature() ? Math.min(lethal, lethalPW) : lethalPW;
-            }
+            int lethal = c.getExcessDamageValue();
```

Comment:
You just need to stick with GameEntity on the parts above here

Also this is no equivalent refactor, the deathtouch part was important:
> 120.4a. First, if an effect that's causing damage to be dealt states that excess damage that would be dealt to a permanent is dealt to another permanent or player instead, the damage event is modified accordingly. If the first permanent is a creature, the excess damage is the amount of damage in excess of what would be lethal damage, taking into account damage already marked on the creature and damage from other sources that would be dealt at the same time. (See rule 120.6.) **Any amount of damage greater than 1 is excess damage if the source dealing that damage to a creature has deathtouch.**

Reply:
Yea I Don't know why IntelliJ changed that. I only had changed this one line.

Reply:
This is what happens when I try to sneak in a fix while my kids are being crazy :(

Reply:
no problem, looks good now